cmake/GlobalConfig.h.in

```diff
@@ -10,7 +10,7 @@
 #cmakedefine US_ENABLE_THREADING_SUPPORT
 #cmakedefine US_HAVE_VISIBILITY_ATTRIBUTE
 
-#define US_DEBUG_POSTFIX "@CMAKE_DEBUG_POSTFIX@"
+#define US_LIB_POSTFIX "@US_LIB_POSTFIX@"
```

Comment:
This is problematic when building using a Visual Studio solution. When this file is generated by CMake it contains an empty string or a "d" (depending on how cmake generated the VS solution). The Visual Studio solution can switch between building debug and release and GlobalConfig.h doesn't get updated accordingly. 

There could be some preprocessor logic which defines ```US_LIB_POSTFIX``` correctly depending on whether its building for debug or release.